www/go/modules/community/filteraddressbook/views/extjs3/MainPanel.js

```diff
@@ -0,0 +1,3 @@
+go.modules.community.filteraddressbook.MainPanel = Ext.extend(go.modules.ModulePanel, {
+	html: "Hello world"
```

Comment:
question: is this used, was this a test, or is it required?

Reply:
This is the basic View file generated with filteraddressbook module. This is not visible at any point of time